CHANGELOG.md

```diff
@@ -1,7 +1,76 @@
-# Thrust 1.11.0 (NVIDIA HPC SDK 20.11)
+# CUB 1.11.0
 
-CUB 1.11.0 is the major release accompanying the NVIDIA HPC SDK 20.11 release.
-It introduces a new radix sort backend with improved performance.
+## Summary
+
+CUB 1.11.0 is a minor release providing bugfixes and performance enhancements.
+
+It includes a new `DeviceRadixSort` backend that improves performance by up to
+2x on supported keys and hardware.
+
+Our CMake package and build system continue to see improvements
+with `add_subdirectory` support, installation rules, status messages, and other
+features that make CUB easier to use from CMake projects.
+
+The release includes several other bugfixes and modernizations, and received
+updates from 11 contributors.
+
+## Breaking Changes
+
+- NVIDIA/cub#201: The intermediate accumulator type used when `DeviceScan` is
+  invoked with different input/output types is now consistent
+  with [P0571](https://wg21.link/P0571). This may produce different results for
+  some edge cases when compared with earlier releases of CUB.
+
+## New Features
+
+- NVIDIA/cub#204: Faster `DeviceRadixSort`, up to 2x performance increase for
+  32/64-bit keys on SM60+. Thanks to Andy Adinets for this contribution.
```

Comment:
Can we say "SM60+ (Pascal and up)"? Some of our users will be familiar with our architectures by name only.

Reply:
Good point, updated.